src/main/requests/page/linked_instagram_account/GetLinkedInstagramAccountResponse.ts

```diff
@@ -0,0 +1,22 @@
+import { AbstractResponse } from '../../AbstractResponse';
+import { LinkedInstagramAccountData } from '../../data/LinkedInstagramAccountData';
+
+
+/**
+ * Class that represents a response from a Get Me request.
+ *
+ * @author Tiago Grosso <tiagogrosso99@gmail.com>
+ * @since 0.3.0
+ */
+export class GetLinkedInstagramAccountResponse extends AbstractResponse<LinkedInstagramAccountData> {
+    /**
+     * Facebook requires Instagram accounts to be linked to a Facebook page and also
```

Comment:
I'd change this to be something like:

"Gets the list of authorized facebook pages." and then add this explanation after that